colosseum/parser.py

```diff
@@ -220,3 +221,65 @@ def quotes(value):
         return Quotes(parsed_values)
 
     raise ValueError('Length of quote items must be a multiple of 2!')
+
+
+##############################################################################
+# Outline shorthand
+##############################################################################
+def _parse_outline_property_part(value, outline_dict):
+    """Parse outline shorthand property part for known properties."""
+    from .constants import (  # noqa
+        OUTLINE_COLOR_CHOICES, OUTLINE_STYLE_CHOICES, OUTLINE_WIDTH_CHOICES,
+    )
+
+    for property_name, choices in {'outline_color': OUTLINE_COLOR_CHOICES,
+                                   'outline_style': OUTLINE_STYLE_CHOICES,
+                                   'outline_width': OUTLINE_WIDTH_CHOICES}.items():
+        try:
+            value = choices.validate(value)
+        except (ValueError, ValidationError):
+            continue
+
+        if property_name in outline_dict:
+            raise ValueError('Invalid duplicated property!')
+
+        outline_dict[property_name] = str(value)  # FIXME: use rgb() on tests?
```

Comment:
Is this FIXME still valid? Is the cast to str needed at all?

Reply:
No, I forgot to fix this, let me push again